internal/policy/verify_test.go

```diff
@@ -141,14 +141,14 @@ func TestVerifyCommit(t *testing.T) {
 	}
 
 	expectedStatus = map[string]string{tagHash.String(): nonCommitMessage}
-	status = VerifyCommit(testCtx, repo, tagHash.String())
+	status = VerifyCommit(testCtx, repo, true, tagHash.String())
 	assert.Equal(t, expectedStatus, status)
 
 	// Add a commit but don't record it in the RSL
 	commitIDs = common.AddNTestCommitsToSpecifiedRef(t, repo, refName, 1, gpgKeyName)
 
 	expectedStatus = map[string]string{commitIDs[0].String(): unableToFindPolicyMessage}
-	status = VerifyCommit(testCtx, repo, commitIDs[0].String())
+	status = VerifyCommit(testCtx, repo, true, commitIDs[0].String())
```

Comment:
IIRC the test helper already has file policies for "1" and "2". Can you add a test where an unauthorized commit modifies these files?

Reply:
I have added a failing test where a commit edits an unauthorized file path. I had to change all the test to not use the flag applyFilePolicies, since it turns out all commits are editing unauthorized file paths. I think this is being set in the at the start of the test when the repo is being created with createTestStatewithPolicy. To create a valid test, either the policy would have to be changed outside of the func, or the function itself would have to be changed.